lang/csharp/src/apache/main/Schema/SchemaName.cs

```diff
@@ -64,7 +64,7 @@ public SchemaName(String name, String space, String encspace)
                 this.EncSpace = encspace;   // need to save enclosing namespace for anonymous types, so named types within the anonymous type can be resolved
             }
 #pragma warning disable CA1307 // Specify StringComparison
-            else if (name.IndexOf('.') == -1)
+            else if (!name.Contains("."))
```

Comment:
This was another msbuild 16.8 warning/error. Use Contains instead of cmparing IndexOf() to -1, to make the code "more readable". It is kinda cool how many new warnings they introduced with 16.8 msbuild/net 5

Reply:
```
error CA2249: Use 'string.Contains' instead of 'string.IndexOf' to improve readability
```

Reply:
I  think I added some description to the actual commit message when those files were checked in, however those are kinda lost when you are going through the diff